### ISSUE TITLE
Allow enemy soldiers to "leave" conversation

### ISSUE THREAD

**Issue by SomethingSimpl** [Sparker95/Vindicta#740]
You can talk with the soldier and make him stop and wait in front of you for what seems like a very long time. Maybe consider the soldier leaving the conversation after some time of "silence" to prevent weird strategies when players engage in a conversation with a single guy and wait until the rest of the patrol leaves to shoot him easily?

**Comment by Sparker95** [comment]
That would be good to handle, I agree, but I think it's a rather low priority task, I don't know if anyone is using these dialogues with enemy patrols at all.

**Comment by SomethingSimpl** [comment]
Yeah, its just a thing I came across when playing with a few friends. Thought Id put it here. No priority on this really.
